src/pages/RoomDetailPage.vue

```diff
@@ -54,6 +64,10 @@ export default {
     removeMe () {
       roomService.removeMe(this.room.qrCode)
     },
+    print () {
+      console.log('print...')
```

Comment:
Is this console.log wanted to be in production or was it there just for development?